orchestration/src/test/java/com/sap/ai/sdk/orchestration/client/OrchestrationUnitTest.java

```diff
@@ -66,6 +75,33 @@ public class OrchestrationUnitTest {
                               .templatingModuleConfig(templatingModuleConfig)))
               .inputParams(Map.of());
 
+  private static final CompletionPostRequest MASKING_CONFIG;
+
+  static {
```

Comment:
personally I'd prefer to have test code and assertion closer together for better readability